### PR TITLE
IBX-4494 - PIM page blocks

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   |  |
| Versions      |  |

Add information on new page blocks

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Comment by dabrt** [comment]
Waiting for 
https://issues.ibexa.co/browse/IBX-4176
https://issues.ibexa.co/browse/IBX-4369
Ready for review if these two tickets don't make it to the release

**Review by ciastektk — APPROVED** [review]

File: docs/content_management/block_reference.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "15-inch laptops" or "Fishing rods".
+- **Setup customer group and catalog matching priority rules** - Create at least one assignment: add a row, then select a customer group and a matching catalog. 
+-	**Default catalog** – Select a catalog that is displayed to end-users who are either 
+anonymous or do not belong to any of the customer groups assigned to specific catalogs.
+-	**Limit** – Set the number of products to be displayed.
+
+On the **Design** tab, in the **View** field, select the layout be used to present a list of products and submit your changes.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:55]:
```suggestion
On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
```

File: docs/content_management/block_reference.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+- In the **URL** box, enter a URL to open when clicking the selected image.
+
+## Catalog block
+
+Before you configure this block, make sure that there are [catalogs](../pim/work_with_catalogs.md) that are defined 
+and published in your PIM.
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "15-inch laptops" or "Fishing rods".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:49]:
```suggestion
- **Name** – Optionally, enter a name for the page block, for example, "15-inch laptops" or "Fishing rods".
```

File: docs/content_management/block_reference.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+- **Setup customer group and catalog matching priority rules** - Create at least one assignment: add a row, then select a customer group and a matching catalog. 
+-	**Default catalog** – Select a catalog that is displayed to end-users who are either
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:51]:
```suggestion
- **Default catalog** – Select a catalog that is displayed to end-users who are either 
```

File: docs/content_management/block_reference.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+Before you configure this block, make sure that there are [catalogs](../pim/work_with_catalogs.md) that are defined 
+and published in your PIM.
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "15-inch laptops" or "Fishing rods".
+- **Setup customer group and catalog matching priority rules** - Create at least one assignment: add a row, then select a customer group and a matching catalog. 
+-	**Default catalog** – Select a catalog that is displayed to end-users who are either 
+anonymous or do not belong to any of the customer groups assigned to specific catalogs.
+-	**Limit** – Set the number of products to be displayed.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:53]:
```suggestion
- **Limit** – Set the number of products to be displayed.
```

File: docs/content_management/block_reference.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+- In the **Name** box, provide the name of the block that will appear on the Page.
+- In the **Folder** section, click **Select content**, navigate through the content, select a folder containing images to display and click **Confirm**.
+After submitting the settings, all images in the folder will appear in the Gallery block.
+Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
+
+## Last purchased block
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "Up-to-date products" or "Your recent purchases".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:138]:
```suggestion
- **Name** – Optionally, enter a name for the page block, for example, "Up-to-date products" or "Your recent purchases".
```

File: docs/content_management/block_reference.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+-	**Personalization scenario** – Select "Last purchased" to display products that were recently purchased from PIM by any user, or "Last purchased by user" to display products that were recently purchased by the current user.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:139]:
```suggestion
- **Personalization scenario** – Select "Last purchased" to display products that were recently purchased from PIM by any user, or "Last purchased by user" to display products that were recently purchased by the current user.
```

File: docs/content_management/block_reference.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+-	**Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Hardware" or "Linear bearings".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:140]:
```suggestion
- **Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Hardware" or "Linear bearings".
```

File: docs/content_management/block_reference.md [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+-	**Limit** – Set the number of products to be displayed.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:141]:
```suggestion
- **Limit** – Set the number of products to be displayed.
```

File: docs/content_management/block_reference.md [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+
+On the **Design** tab, in the **View** field, select the layout be used to present a list of products and submit your changes.
+
+## Last viewed block
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "Popular products" or "Your browsing history".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:149]:
```suggestion
- **Name** – Optionally, enter a name for the page block, for example, "Popular products" or "Your browsing history".
```

File: docs/content_management/block_reference.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+-	**Personalization scenario** – Select "Last viewed" to display products that were recently viewed by any user, or "Last viewed by user" to display products that were recently viewed by the current user.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:150]:
```suggestion
- **Personalization scenario** – Select "Last viewed" to display products that were recently viewed by any user, or "Last viewed by user" to display products that were recently viewed by the current user.
```

File: docs/content_management/block_reference.md [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+-	**Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Bicycles" or "Dental care".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:151]:
```suggestion
- **Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Bicycles" or "Dental care".
```

File: docs/content_management/block_reference.md [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+-	**Limit** – Set the number of products to be displayed.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:152]:
```suggestion
- **Limit** – Set the number of products to be displayed.
```

File: docs/content_management/block_reference.md [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+
+On the **Design** tab, in the **View** field, select the layout be used to present a list of products and submit your changes.
+
+## Personalized block
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Name** – Optionally, enter a name for the page block, for example, "Bestsellers".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:160]:
```suggestion
- **Name** – Optionally, enter a name for the page block, for example, "Bestsellers".
```

File: docs/content_management/block_reference.md [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+-	**Select a scenario** – Select "Landing page" or "Top clicked".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:161]:
```suggestion
- **Select a scenario** – Select "Landing page" or "Top clicked".
```

File: docs/content_management/block_reference.md [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+-	**Select a Content Type...** – Select "Product".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:162]:
```suggestion
- **Select a Content Type...** – Select "Product".
```

File: docs/content_management/block_reference.md [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+-	**Display limit** – Set the number of products to be displayed, for example, 4.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:163]:
```suggestion
- **Display limit** – Set the number of products to be displayed, for example, 4.
```

File: docs/content_management/block_reference.md [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+On the **Design** tab, in the **View** field, change the layout to "Products" and submit your changes.
+
+## Product collection block
+
+On the **Basic** tab, perform the following actions:
+
+- In the **Name** box, provide the name of the block that will appear on the Page, 
+for example, "Summer giveway".
+- In the **Product list** area, create a list of products: 
+    - In the **Add product code** field, enter a comma separated list of product/variant codes and click **Add**.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:174]:
```suggestion
- In the **Add product code** field, enter a comma separated list of product/variant codes and click **Add**. 
```

File: docs/content_management/block_reference.md [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+
+## Product collection block
+
+On the **Basic** tab, perform the following actions:
+
+- In the **Name** box, provide the name of the block that will appear on the Page, 
+for example, "Summer giveway".
+- In the **Product list** area, create a list of products: 
+    - In the **Add product code** field, enter a comma separated list of product/variant codes and click **Add**. 
+    - Click **Select product**. Then, in content browser, select products and click **Confirm**.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:175]:
```suggestion
- Click **Select product**. Then, in content browser, select products and click **Confirm**.
```

File: docs/content_management/block_reference.md [hunk 19]
```diff
@@ -0,0 +1,234 @@
...
+
+    Due to a technical limitation, content browser does not display product variants.
+
+On the **Design** tab, in the **View** field, select the layout be used to present a list of products and submit your changes.
+
+## Recently added block
+
+On the **Basic** tab, set values in the following fields:
+
+-	**Block name** – Optionally, enter a name for the page block, for example, "Newest additions".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:187]:
```suggestion
- **Block name** – Optionally, enter a name for the page block, for example, "Newest additions".
```

File: docs/content_management/block_reference.md [hunk 20]
```diff
@@ -0,0 +1,234 @@
...
+-	**Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Personal hygiene" or "Drill bits".
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:188]:
```suggestion
- **Product Types to be displayed** – Select the type of products to be displayed on the list, for example, "Personal hygiene" or "Drill bits".
```

File: docs/content_management/block_reference.md [hunk 21]
```diff
@@ -0,0 +1,234 @@
...
+-	**Display limit** – Set the number of products to be displayed, for example, 4.
```
Comment (julitafalcondusza) [inline docs/content_management/block_reference.md:189]:
```suggestion
- **Display limit** – Set the number of products to be displayed, for example, 4.
```

**Review by julitafalcondusza — APPROVED** [review]

**Pull request merged**